Assets/MixedRealityToolkit.SDK/Experimental/Features/Dwell/Samples/CustomDwellHandler.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
+{
+    /// <summary>
+    /// Example to demonstrate override
+    /// </summary>
+    public class CustomDwellHandler : DwellHandler
+    {
+        public override float CalculateDwellProgress()
```

Comment:
Can't use this syntax since some MRTK projects do not use c#7 yet :(

---
In reply to: [313156541](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5594#discussion_r313156541) [](ancestors = 313156541)

Reply:
Is MRTK trying to support <= Unity 2018.3? Everything 2018.3+ supports C# 7.

---
In reply to: [316857577](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5594#discussion_r316857577) [](ancestors = 316857577,313156541)